source/_integrations/discovery.markdown

```diff
@@ -110,7 +110,13 @@ If mDNS is still not working:
 #### mDNS forwarding
 If it's not possible to have Home Assistant and the devices on the same network, mDNS forwarding may allow mDNS discovery between networks.
 
-mDNS forwarding is a configurable option in some routers. It can also be called mDNS reflector or mDNS repeater, depending on the manufacturer.
+mDNS forwarding is a configurable option in some routers. It can also be called mDNS reflector or mDNS repeater, depending on the manufacturer. For Docker setups that use bridge network (instead of `net=host` - not recommended) the following Avahi setting can help:
```

Comment:
Home Assistant requires the host network, as a matter of fact, running your container without setting that = not supported.


See the Architecture Decision Record on this subject:

> The only supported way to run the container is on the host network as root with full privileges.

<https://github.com/home-assistant/architecture/blob/master/adr/0013-home-assistant-container.md#decision>

Reply:
Sorry - I was unaware of that. I tried to use net=host for a long time, however certain docker features (such as reverse proxies) only work in the default bridge mode. Using this mDns reflector setting was the only change I had to make in order to have Home Assistant fully working in bridge mode, but it took a really long time to find it in the forum.

In any case - please close / remove the PR in this case.